.github/workflows/nodejs.yml

```diff
@@ -6,8 +6,7 @@ on:
       - main
       - 'dependabot/**'
   pull_request:
-    branches:
-      - '**'
+  merge_group:
```

Comment:
[note] Try a [merge queue](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/managing-a-merge-queue).